airflow/contrib/hooks/gcp_spanner_hook.py

```diff
@@ -39,13 +37,14 @@ def __init__(self,
                  delegate_to=None):
         super(CloudSpannerHook, self).__init__(gcp_conn_id, delegate_to)
 
+    @GoogleCloudBaseHook.fallback_to_default_project_id
     def get_client(self, project_id):
-        # type: (str) -> Client
+        # noinspection LongLine
```

Comment:
Can we remove this, please? It would also remove real long-line code

Reply:
Removed all those, though I wish there was a more Intellij/PEP8/flake consistent way of pasting long URLs in docstrings :(